CHANGELOG.md

```diff
@@ -1,3 +1,355 @@
+## 2.0.0 (2024-03-20)
```

Comment:
Wow! I don't know why it generates this CHANGELOG, because you only removed the CSS import , so nothing here makes sense. The tool generating it must be wrong.

Nevermind, we'll go ahead as we'll replace this changelog generation in the near future.